### PR TITLE
Updating details for support of DLP policy tips @chrfox

### PR DESCRIPTION
**Pull request by shekharpalta**
@chrfox  Please review the above changes and see if it needs any edits

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/use-notifications-and-policy-tips.md [hunk 1]
```diff
@@ -188,6 +188,102 @@ By default, policy tips display text similar to the following for an item on a s
 
 You can customize the text for policy tips separately from the email notification. Unlike custom text for email notifications (see above section), custom text for policy tips does not accept HTML or tokens. Instead, custom text for policy tips is plain text only with a 256-character limit.
   
+## Support Matrix for DLP policy tips across Microsoft apps
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/use-notifications-and-policy-tips.md:191]:
```suggestion
## Support matrix for DLP policy tips across Microsoft apps
```
```suggestion
## Support Matrix for DLP policy tips across Microsoft apps
```

File: microsoft-365/compliance/use-notifications-and-policy-tips.md [hunk 2]
```diff
@@ -188,6 +188,102 @@ By default, policy tips display text similar to the following for an item on a s
...
+
+
+|**App and platform**|**DLP policy tip support**|**Sensitive information types supported**|**Conditions and exceptions supported**|**Comments**|
+|--------------------|--------------------------|-----------------------------------------|------------------------------------|------------|
+|Outlook Web Access <br/> |Yes <br/> |All <br/> |Subset  <br/> |See  [DLP predicates supported for DLP policy tips in Outlook Web Access](#dlp-predicates-supported-for-dlp-policy-tips-in-outlook-web-access)  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/use-notifications-and-policy-tips.md:196]:
```suggestion
|Outlook on the web <br/> |Yes <br/> |All <br/> |Subset  <br/> |See  [DLP predicates supported for DLP policy tips in Outlook on the web](#dlp-predicates-supported-for-dlp-policy-tips-in-outlook-web-access)  <br/> |
```
```suggestion
|Outlook Web Access <br/> |Yes <br/> |All <br/> |Subset  <br/> |See  [DLP predicates supported for DLP policy tips in Outlook Web Access](#dlp-predicates-supported-for-dlp-policy-tips-in-outlook-web-access)  <br/> |
```

File: microsoft-365/compliance/use-notifications-and-policy-tips.md (hunk 1 above)
Comment (JohanFreelancer9) [inline microsoft-365/compliance/use-notifications-and-policy-tips.md:191]:
```suggestion
## Support matrix for DLP policy tips across Microsoft apps
```
```suggestion
## Support Matrix for DLP policy tips across Microsoft apps
```

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/compliance/use-notifications-and-policy-tips.md [hunk 3]
```diff
@@ -188,6 +188,102 @@ By default, policy tips display text similar to the following for an item on a s
...
+|Outlook Desktop (Outlook 2013 and beyond) <br/> |Yes <br/> |Subset <br/> |Subset  <br/> |See [Outlook 2013 and later supports showing policy tips for only some sensitive information types](#outlook-2013-and-later-supports-showing-policy-tips-for-only-some-sensitive-information-types) and [Outlook 2013 and later supports showing policy tips for only some conditions and exceptions](#outlook-2013-and-later-supports-showing-policy-tips-for-only-some-conditions-and-exceptions) for details on support for sensitive information types and DLP conditions and actions supported for showing DLP policy tips on Outlook Desktop. <br/> |
+|Outlook Mobile (iOS, Android)/Outlook Mac <br/> |No <br/> |None <br/> |None <br/> |DLP policy tips are not supported on Outlook mobile <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/use-notifications-and-policy-tips.md:198]:
```suggestion
|Outlook for mobile <br/> |No <br/> |None <br/> |None <br/> |DLP policy tips are not supported on Outlook for mobile <br/> |
```

**Review by JohanFreelancer9 — COMMENTED** [review]
@yogkumgit @shekharpalta - Looking at the doc I have noticed a number of inconsistent/ incorrect product names that do not follow the current naming conventions, for example, Outlook Mobile (should be Outlook for mobile), One Drive for Business (should only be OneDrive), etc. However, I would rather wait until @chrfox has reviewed the content before I do a copy review. Thanks.

**Comment by yogkumgit** [comment]
@chrfox Please review/merge

**Comment by chrfox** [comment]
unusable as is, moved this content to a stand alone reference topic

**Pull request closed without merging**